nemo/collections/common/parts/preprocessing/manifest.py

```diff
@@ -77,6 +77,7 @@ def item_iter(
 
 def __parse_item(line: str, manifest_file: str) -> Dict[str, Any]:
     item = json.loads(line)
+    manifest_path, _ = split(manifest_file)
```

Comment:
Your dataset need not be in the same location as your manifest, this change is incorrect and would give a vague error if this assumption is invalid c